nautobot/dcim/models/devices.py

```diff
@@ -1331,7 +1329,7 @@ def clean(self):
     "graphql",
     "webhooks",
 )
-class ControllerDeviceGroup(TreeModel, PrimaryModel, ConfigContextModel):
+class ControllerManagedDeviceGroup(TreeModel, PrimaryModel, ConfigContextModel):
```

Comment:
Remove `ConfigContextModel` mixin from both  `Controller` and `ControllerManagedDeviceGroup` per today's discussion.

Reply:
Removed